src/frontend/components/Felleskomponenter/BlokkerTilbakeKnappModal/BlokkerTilbakeKnappModal.tsx

```diff
@@ -23,14 +23,17 @@ const Flex = styled.div`
     align-items: center;
     gap: 1rem;
 `;
-const StyledLenke = styled(Lenke)`
+const TekstBlockMarginHøyre = styled(TekstBlock)`
```

Comment:
Denne stylingen tror jeg kan fjernes. Både fordi ModalContent tar seg av den nå, pluss at jeg er usikker på om den har noen effekt? da vi ikke har lagt inn støtte for classnames på TekstBlock som jeg tror vi må ha for at Styled components skal funke på den 🤔

Reply:
Fiksa i [Fjern styling](https://github.com/navikt/familie-ks-soknad/pull/108/commits/2d6734bb3e309e6165f73499573b86209e5fa639)